vscode/webviews/AuthPage.tsx

```diff
@@ -280,7 +285,7 @@ const ClientSignInForm: React.FC<ClientSignInFormProps> = memo(
             isSubmitting: false,
             showAuthError: false,
             formData: {
-                endpoint: authStatus?.endpoint ?? '',
+                endpoint: authStatus && !isDotCom(authStatus) ? authStatus.endpoint : '',
```

Comment:
```suggestion
                endpoint: authStatus && !isDotCom(authStatus) ? authStatus.endpoint : '',
```

If the user has just signed out of dotcom or have never signed in before, would the endpoint in their authstatus set to DotCom by default?

maybe we could also check if the user's available endpoints (endpoint history) contain dotcom or not?

Reply:
> If the user has just signed out of dotcom or have never signed in before, would the endpoint in their authstatus set to DotCom by default?

These two cases might be different, but in the completely new setup case we *do* default the endpoint to dotcom. Today we display `https://sourcegraph.com` and this is effective at suppressing it as a default. I have updated the test plan to cover the "new install" case (which I'm simulating with a new user data dir & profile temp.)

> maybe we could also check if the user's available endpoints (endpoint history) contain dotcom or not?

Yeah, great idea. Let's do it in a follow up.